pkg/service/well-known/did_configuration.go

```diff
@@ -1,24 +1,47 @@
 package wellknown
 
 import (
+	"bytes"
 	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
 	"time"
 
 	"github.com/TBD54566975/ssi-sdk/credential"
 	"github.com/TBD54566975/ssi-sdk/credential/integrity"
 	"github.com/TBD54566975/ssi-sdk/did"
+	"github.com/TBD54566975/ssi-sdk/did/resolution"
+	"github.com/goccy/go-json"
 	"github.com/pkg/errors"
 	credint "github.com/tbd54566975/ssi-service/internal/credential"
+	"github.com/tbd54566975/ssi-service/internal/util"
 	svcframework "github.com/tbd54566975/ssi-service/pkg/service/framework"
 	"github.com/tbd54566975/ssi-service/pkg/service/keystore"
+	"github.com/tbd54566975/ssi-service/pkg/service/schema"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
 )
 
 type DIDConfigurationService struct {
 	keyStoreService *keystore.Service
+	validator       *credint.Validator
+
+	HTTPClient *http.Client
 }
 
-func NewDIDConfigurationService(keyStoreService *keystore.Service) *DIDConfigurationService {
-	return &DIDConfigurationService{keyStoreService: keyStoreService}
+func NewDIDConfigurationService(keyStoreService *keystore.Service, didResolver resolution.Resolver, schema *schema.Service) (*DIDConfigurationService, error) {
+	client := &http.Client{Transport: otelhttp.NewTransport(http.DefaultTransport)}
```

Comment:
should this be the default client (with timeouts, etc) and then add in the otel?

Reply:
DefaultClient is the same as the zero value of `http.Client`. Timeouts are carried by the `ctx` when the call is made. 

We _could_ have an `httpClientProvider`, but then that get's into the dependency injection and I wanted to keep this simple.